backend/app/routes.py

```diff
@@ -209,10 +209,10 @@ def aggregate_travel_times(start_node, end_node, start_time, end_time, start_dat
             JOIN congestion.network_segments_daily AS cn USING (segment_id)
             WHERE   
                 cn.hr <@ %(time_range)s::numrange
-                AND date_part('isodow', cn.dt)::integer <@ '[1, 5]'::int4range
+                AND date_part('isodow', cn.dt)::integer IN %(dow_list)s
                 AND cn.dt <@ %(date_range)s::daterange
                 AND NOT EXISTS (
-                    SELECT 1 FROM ref.holiday WHERE cn.dt = holiday.dt -- excluding holidays
+                    SELECT %(incl_holiday)s FROM ref.holiday WHERE cn.dt = holiday.dt -- excluding holidays

```

Comment:
This won't have the effect you're expecting. See https://www.postgresql.org/docs/current/functions-subquery.html#FUNCTIONS-SUBQUERY-EXISTS

Reply:
closing, holiday moved to new pr